### PR TITLE
Phase2 Doc update

### PR DESCRIPTION
**Pull request by rnovikovP**
Update doc according to https://github.com/percona-platform/checked/pull/425 

Text version of all checks. 
We must keep this list until we implement an in-product presentation (Planned for 2.34)

### CODE REVIEW

**Comment by rnovikovP** [comment]
@Tusamarco, please check the sanity of the changes. 
@catalinaadam, please check end edit the styles as I've extracted this data from checks, and styles are different(

File: docs/checks.md [hunk 1]
```diff
@@ -40,48 +40,66 @@ Percona Platform hosts all Advisor checks for MySQL, MongoDB, and PostgreSQL dat
...
+| **MySQL index sizes** | Warns if any have indexes larger than data.  This indicates sub-optimial schema and should be reviewed. | Paid |
+| **MySQL replication configuration check** |  Checks if a replica is safely logging replicated transactions.| Paid |
+| **MySQL Users With Granted Public Networks Access** | Notifies about MySQL accounts allowed to be connected from public networks. | Paid |
+| **MySQL Secure Transport** | Warns if MySQL server allows unencrypted remote connections. | Registered, Paid |
+| **MySQL User Check** | Runs a high-level check on user setup | Registered |
 | **MySQL Advanced User Check** | Runs a detailed check on user setup | Paid |
-| **MySQL Security Check** | Runs advisor checks on replica account and password policy. | Paid |
+| **MySQL Security Check for password policy** | Runs advisor checks on password policy. | Paid |
+| **MySQL Security Check for remplication** | Runs advisor checks on replica account. | Paid |
+| **MySQL Replication privileges** | Warns if replication privileges is mixed with more elevated privileges. | Paid |
```
Comment (catalinaadam) [inline docs/checks.md:60]:
```suggestion
| **MySQL Replication privileges** | Warns if replication privileges are mixed with more elevated privileges. | Paid |
```

File: docs/checks.md [hunk 2]
```diff
@@ -40,48 +40,66 @@ Percona Platform hosts all Advisor checks for MySQL, MongoDB, and PostgreSQL dat
...
+| **MySQL tables without Primary Key** | Warns if  tables added without primary keys. | Paid |
 | **MySQL Test Database** | This check returns a notice if there is database with name ‘test’ or ‘test\_%’. | Registered, Paid |
```
Comment (catalinaadam) [inline docs/checks.md:62]:
```suggestion
| **MySQL Test Database** | Notifies of any databases named **‘test’** or **‘test\_%’**. | Registered, Paid |
```

File: docs/checks.md [hunk 3]
```diff
@@ -40,48 +40,66 @@ Percona Platform hosts all Advisor checks for MySQL, MongoDB, and PostgreSQL dat
...
 | Check Name | Description | Tier |
 | :--------- | :---------- | :--- |
 | **MongoDB Active vs Available Connections** | Checks the ratio between Active and Available connections. | Registered, Paid |
-| **MongoDB Authentication** | Warns if MongoDB authentication is disabled. | Anonymous, Registered, Paid |
-| **MongoDB Security AuthMech** | Warns if MongoDB is not using the default SHA-256 hashing as SCRAM authentication method. | Paid |
-| **MonogDB IP Bindings** | Warns if MongoDB network binding is not set as recommended. | Anonymous, Registered, Paid |
+| **MongoDB Authentication** | Warns if MongoDB authentication is disabled. | Registered, Paid |
+| **MongoDB Security AuthMech** | Warns if MongoDB is not using the default SHA-256 hashing as SCRAM authentication method. | Registered, Paid |
+| **MonogDB IP Bindings** | Warns if MongoDB network binding is not set as recommended. | Paid |
+| **MonogDB CPU cores** | Warns if the number of CPU cores does not meet the minimum recommended requirements according to best practices | Registered, Paid |
```
Comment (catalinaadam) [inline docs/checks.md:74]:
```suggestion
| **MonogDB CPU cores** | Warns if the number of CPU cores does not meet the minimum recommended requirements according to best practices. | Registered, Paid |
```

File: docs/checks.md [hunk 4]
```diff
@@ -40,48 +40,66 @@ Percona Platform hosts all Advisor checks for MySQL, MongoDB, and PostgreSQL dat
...
+| **MySQL Checks based on values of MySQL configuration variables** | Checks the following settings: **sync_binlog**, **log_bin, general_log**, **tmp_table_size**, **sql_mode**, **read_only**. | Registered,Paid |
+| **MySQL Checks based on values of MySQL replication configuration variables** | Warns if replication is not configured correctly. | Registered,Paid |
 | **MySQL Binary Logs checks, Local infile and SQL Mode checks** | Warns about non-optimal settings for Binary Log, Local Infile and SQL mode. | Registered, Paid |
-| **MySQL Configuration Check** | Warns if parameters not following Percona best practices, for infile, replication threads and replica checksum. | Paid |
-| **MySQL Users With Granted Public Networks Access** | Notifies about MySQL accounts allowed to be connected from public networks. | Registered, Paid |
-| **MySQL User Check** | Runs a high-level check on user setup | Registered, Paid |
+| **MySQL Configuration Check** | Warns if parameters not following Percona best practices, for infile, replication threads and replica checksum. | Registered, Paid |
+| **MySQL InnoDB password lifetime** | Warns about password lifetime. | Registered, Paid |
```
Comment (catalinaadam) [inline docs/checks.md:50]:
```suggestion
| **MySQL InnoDB password lifetime** | Warns if password expiration is not set and users' passwords have an infinite lifetime. | Registered, Paid |
```

File: docs/checks.md [hunk 5]
```diff
@@ -40,48 +40,66 @@ Percona Platform hosts all Advisor checks for MySQL, MongoDB, and PostgreSQL dat
...
 
 ### PostgreSQL checks
 
 | Check Name | Description | Tier |
 | :--------- | :---------- | :--- |
-| **PostgreSQL Archiver is Failing** | Verifies if the archiver has failed. | Paid |
-| **PostgreSQL Cache Hit Ratio** | Checks database hit ratio and complains when this is too low. | Paid |
+| **PostgreSQL Archiver is Failing** | Verifies if the archiver has failed. | Registered, Paid |
+| **PostgreSQL Cache Hit Ratio** | Checks database hit ratio and complains when this is too low. | Registered, Paid |
 | **PostgreSQL Configuration Change Requires Restart/Reload** | Warns when a configuration was changed and requires a server restart/reload | Registered, Paid |
```
Comment (rnovikovP) [inline docs/checks.md:94]:
```suggestion
| **PostgreSQL Configuration Change Requires Restart/Reload** | Warns when a configuration was changed and requires a server restart/reload. | Registered, Paid |
```

**Review by Tusamarco — APPROVED** [review]
Seems good to me

**Pull request merged**